storage/storage.cpp

```diff
@@ -50,6 +50,11 @@ storage::error_msg(storage_base::result error_id)
         case storage_base::result::exists:
             message = "exists";
             break;
+        case storage_base::result::value_too_large:
+            message = "value exceeds ";
```

Comment:
No need to break this into three lines:

mesage = "value exceeds " + std::to_string(MAX_VALUE_SIZE) + "  kb" 

should work fine.

If you stick with this then please put the code in a braces such as:

case ASDF:
{
    ...
    break;
}

Reply:
I have combined the code into one line.